CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing to Spotlight
+
+Spotlight is a collaborative, open source project produced by developers, designers, product owners from several organizations. Spotlight mostly, uses the [contribution model](https://github.com/projectblacklight/blacklight/blob/master/CONTRIBUTING.md) from it's parent project [Blacklight](https://github.com/projectblacklight/blacklight). There are a few differences from the Blacklight process that this document will highlight.
+
+## User experience changes
+
+Spotlight has been designed by a team of user experience designers. While there are a lot of great opportunities to improve the user experience, some contributed changes might change other aspects of user experience that has been accounted. When contributing updates or changes to the Spotlight user experience here are some useful guidelines to follow.
+ - Determine if there is a way to contribute this in a backwards compatible or configurable way
+ - [Open an issue](https://github.com/projectblacklight/spotlight/issues/new) on the project and engage with core UX designers and developers
+ - Contribute user stories to an issue or engage with the Spotlight community through the [Blacklight Development Google group](https://groups.google.com/forum/#!forum/blacklight-development)
+
+
+## Other notes
+
+Because Spotlight is a large codebase being maintained over several years, we encourage some additional guidelines around contributing.
```

Comment:
> Because Spotlight is a large codebase being maintained over several years,

Because Spotlight is a large codebase that has been developed and maintained over several years,

- Replaced "being maintained" with "that has been developed and maintained"